tests/ember-electron/main.js

```diff
@@ -47,15 +47,3 @@ app.on('ready', function onReady() {
     mainWindow = null;
   });
 });
-
-let cwd = __dirname || resolve(dirname());
```

Comment:
This is in the main process, and the other is in the renderer process, and because of the workaround for module include paths in the render process' shim, they are fairly different setups that I think need to be tested independently.

I entirely agree that we could probably have better and more thorough ways of testing this main-process functionality and ensuring that the package is assembled correctly and whatnot, but this is what we have right now, and it's been really really helpful to me as a smoke test when making changes to the addon, so I'd really like to keep it until we have something that replaces it.